PackageInfo.g

```diff
@@ -68,13 +84,64 @@ Persons := [
     Place         := "Aachen",
     Institution   := "Lehrstuhl D für Mathematik, RWTH Aachen",
   ),
+rec(
+    LastName      := "Hähndel",
+    FirstNames    := "Paula",
+    IsAuthor      := false,
+    IsMaintainer  := false,
+    Email         := "paula.haehndel@mathematik.uni-halle.de",
+    WWWHome       := "https://algebra.mathematik.uni-halle.de/haehndel/",
+    PostalAddress := Concatenation( [
+                       "Paula Hähndel\n",
+                       "Georg-Cantor-Haus\n",
+                       "Theodor-Lieser-Str. 5\n",
+                       "06120 Halle (Saale)\n",
+                       "Germany" ] ),
+    Place         := "Halle (Saale)",
+    Institution   := "Martin Luther University of Halle-Wittenberg"
+  ),
+  rec(
+    LastName      := "Hulpke",
+    FirstNames    := "Alexander",
+    IsAuthor      := false,
+    IsMaintainer  := false,
+    Email         := "hulpke@colostate.edu",
+    WWWHome       := "https://www.math.colostate.edu/~hulpke/",
+    PostalAddress := Concatenation( [
+                       "Alexander Hulpke\n",
+                       "Department of Mathematics\n",
+                       "Colorado State University\n",
+                       "1874 Campus Delivery\n",
+                       "Fort Collins, CO 80523-1874\n",
+                       "USA" ] ),
+    Place         := "Fort Collins",
+    Institution   := "Colorado State University"
+  ),
   rec(
     LastName      := "Howe",
     FirstNames    := "Stephen",
     IsAuthor      := false,
     IsMaintainer  := false,
     PostalAddress := "Unknown",
   ),
+  rec(
+    LastName      := "Jefferson",
+    FirstNames    := "Christopher",
+    IsAuthor      := false,
+    IsMaintainer  := false,
+    Email         := "caj21@st-andrews.ac.uk",
+    WWWHome       := "https://caj.host.cs.st-andrews.ac.uk",
+    PostalAddress := Concatenation( [
+                       "Christopher Jefferson\n",
+                       "Jack Cole Building\n",
+                       "Colorado State University\n",
+                       "North Haugh\n",
+                       "St. Andrews\n",
```

Comment:
The name of the town “St Andrews” does not have a full-stop in it, at least in my opinion and the university’s option. The story is that the name of the town was fixed as “St Andrews” before the concept of denoting abbreviation by a . was even invented.

So please remove the full stop from this line and from the place and institution fields. (cf. Steve Linton’s data)